qiita_db/metadata_template/base_metadata_template.py

```diff
@@ -54,6 +54,22 @@
 # information
 QIITA_COLUMN_NAME = 'qiita_sample_column_names'
 
+INSDC_NULL_VALUES = {
+    'not collected': 'not collected',
+    'not provided': 'not provided',
+    'restricted access': 'restricted access',
+    'not applicable': 'not applicable',
+    'unspecified': 'not applicable',
```

Comment:
Just to confirm, unspecified should be mapped to non-applicable? It seems to be the only one without a unique string.

Reply:
yup, long discussion on qiita.admin ...